helios-tools/src/main/java/com/spotify/helios/cli/command/HostListCommand.java

```diff
@@ -90,6 +91,10 @@ public HostListCommand(final Subparser parser) {
         .action(storeTrue())
         .help("Print full host names.");
 
+    statusArg = parser.addArgument("--status")
+        .nargs("?")
+        .help("Filter hosts by its status, i.e. \"UP\" or \"DOWN\".");
```

Comment:
You might want to checkout out the built-in choices() method. E.g.

    .choices(Collections2.transform(
        Arrays.asList(HostStatus.Status.values()), new Function<HostStatus.Status, String>() {
          @Override
          public String apply(final HostStatus.Status input) {
            return input.toString();
          }
        }).toArray(new String[0]));

(it's icky in java 7, much nicer in java 8)